drivers/scheduler/k8s/specs/elasticsearch-crd-webhook/aws/aws-storage-class.yaml

```diff
@@ -0,0 +1,8 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: elasticsearch-sc
+provisioner: kubernetes.io/aws-ebs
```

Comment:
Can we use the ebs csi provisioner `ebs.csi.aws.com` ? 

Link: https://docs.aws.amazon.com/eks/latest/userguide/ebs-sample-app.html

Reply:
⚠️ I think I got it right? should the parameters still remain?

Reply:
✅